database/mssql/scripts/versions/v_41_ddl.sql

```diff
@@ -0,0 +1,250 @@
+      ALLOW_PAGE_LOCKS = ON
+      ) ON [PRIMARY]
+   ) ON [PRIMARY];
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_CREATE_USERID] DEFAULT(user_name())
+FOR [DB_CREATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_CREATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_CREATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_LAST_UPDATE_USERID] DEFAULT(user_name())
+FOR [DB_LAST_UPDATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_LAST_UPDATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_LAST_UPDATE_TIMESTAMP]
+
+-- Check Contraints
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] WITH CHECK ADD  CONSTRAINT DK_ORBC_LOA_DETAILS_IS_ACTIVE_VAL CHECK ([IS_ACTIVE] IN ('Y','N'));
+GO
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_REVISION]  DEFAULT ((0)) FOR [REVISION]
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_PREVIOUS_REVISION]  DEFAULT ((0)) FOR [PREVIOUS_LOA_ID]
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_ORIGINAL_ID]  DEFAULT ((0)) FOR [ORIGINAL_LOA_ID]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS]  WITH CHECK ADD  CONSTRAINT [FK_ORBC_LOA_DETAILS_COMPANY] FOREIGN KEY([COMPANY_ID])
+REFERENCES [dbo].[ORBC_COMPANY] ([COMPANY_ID])
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] CHECK CONSTRAINT [FK_ORBC_LOA_DETAILS_COMPANY]
+
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_PERMIT_TYPE_DETAILS_DB_CREATE_USERID] DEFAULT(user_name())
+FOR [DB_CREATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_PERMIT_TYPE_DETAILS_DB_CREATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_CREATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_PERMIT_TYPE_DETAILS_DB_LAST_UPDATE_USERID] DEFAULT(user_name())
+FOR [DB_LAST_UPDATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_PERMIT_TYPE_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_PERMIT_TYPE_DETAILS_DB_LAST_UPDATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_LAST_UPDATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES] ADD CONSTRAINT [DF_ORBC_LOA_VEHICLES_DB_CREATE_USERID] DEFAULT(user_name())
+FOR [DB_CREATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES] ADD CONSTRAINT [DF_ORBC_LOA_VEHICLES_DB_CREATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_CREATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES] ADD CONSTRAINT [DF_ORBC_LOA_VEHICLES_DB_LAST_UPDATE_USERID] DEFAULT(user_name())
+FOR [DB_LAST_UPDATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES] ADD CONSTRAINT [DF_ORBC_LOA_VEHICLES_DB_LAST_UPDATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_LAST_UPDATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_VEHICLES]
+   WITH CHECK ADD CONSTRAINT [ORBC_LOA_VEHICLES_LOA_ID_FK] FOREIGN KEY ([LOA_ID]) REFERENCES [permit].[ORBC_LOA_DETAILS]([LOA_ID])  ON DELETE CASCADE
```

Comment:
@john-fletcher-aot  Should we cascade on delete as part of DB constraints or handle it in code?